src/client/mockttp-client.ts

```diff
@@ -153,6 +153,17 @@ export default class MockttpClient extends AbstractMockttp implements Mockttp {
         }
     }
 
+    async pendingMocks(): Promise<any> {
```

Comment:
This should return a specific type, not `any`. You probably want `Promise<string[]>`